README.md

```diff
@@ -379,6 +382,12 @@ Go ahead, deploy `osv-microservice-demo` application:
 $ kubectl create -f ./virtlet_deploy/micro-services.yaml
 
 // Deploy k8s Deployment definitions for osv-microservice-demo
+$ kubectl create -f ./virtlet_deploy/micro-deployments-local.yaml
+```
+
+Or to use prebuilt unikernels that are available on S3 object store:
```

Comment:
I agree with your second suggestion: I've described how to deploy application using pre-composed S3 unikernels here in main README and extracted documentation on how to compose unikernels on your own in auxilary file.

Turns out, however, that the latter option (i.e. deploying your own unikernels) became quite cumbersome lately since Virtlet is now only able to download images using `https` connection. I've opened an [issue](https://github.com/Mirantis/virtlet/issues/292) for this.